packages/zoe/src/contracts/helpers/bondingCurves.js

```diff
@@ -6,23 +6,23 @@ import { natSafeMath } from './safeMath';
 
 const { add, subtract, multiply, floorDivide } = natSafeMath;
 
-export const makeConstProductBC = (zoe, assays) => {
-  const unitOpsArray = zoe.getUnitOpsForAssays(assays);
+export const makeConstProductBC = (zoe, issuers) => {
+  const amountMathArray = zoe.getAmountMathForIssuers(issuers);
   return harden({
     /**
-     * Contains the logic for calculating how many units should be given
+     * Contains the logic for calculating how many amount should be given
      * back to the user in exchange for what they sent in. It also
-     * calculates the new units of the assets in the pool. Reused in
+     * calculates the new amount of the assets in the pool. Reused in
      * several different places, including to check whether an offer is
      * valid, getting the current price for an asset on user request, and
      * to do the actual reallocation after an offer has been made.
-     * @param  {units[]} poolUnitsArray - an array of the current units in the
+     * @param  {amount[]} poolAmountsArray - an array of the current amount in the
      * liquidity pool
-     * @param  {units} unitsIn - the units sent in by a user
+     * @param  {amount} amountIn - the amount sent in by a user
      * @param  {number} feeInTenthOfPercent=3 - the fee taken in tenths of
-     * a percent. The default is 0.3%. The fee is taken from unitsIn
+     * a percent. The default is 0.3%. The fee is taken from amountIn
      */
-    getPrice: (poolUnitsArray, unitsIn, feeInTenthOfPercent = 3) => {
+    getPrice: (poolAmountsArray, amountIn, feeInTenthOfPercent = 3) => {
```

Comment:
Can we calculate in 'basis points'  (== .01 percent) instead of 'tenthOfPercent'? Anyone who works in finance will be much happier with that.  It would mean the fee is 30 basis points, but I think people who had contact with banking or wall street will find that much more natural, and others won't find it hard to understand.

Reply:
I really like that. I'll make an issue outside of this PR.

Reply:
Issue made https://github.com/Agoric/agoric-sdk/issues/564